conf/messages

```diff
@@ -25,6 +25,8 @@ caption = Transport details
 
 checkYourAnswers.heading = Check your answers
 checkYourAnswers.title = Check your answers - Transport details
+transportMeans.departure.heading = Check your answers
```

Comment:
The 
Do you want to add the type of identification?
should be
Do you want to add the type of identification for the departure means of transport?